### PR TITLE
remove ouroboros proxy configs

### PR DESCRIPTION
**Pull request by camallen**
cleanup the old configs for custom ourboros sites to connect to API via custom paths on their deployed domains (avoid CORS issues etc) 

@adammcmaster if this is ok to remove then i'll keep cleaning up the codebase. Are there any other configs we can cleanup / remove?

### CODE REVIEW

File: nginx-api-proxy.conf [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-    # Put the URL in a variable to force re-resolution of the DNS name
-    # Otherwise nginx will cache it indefinitely and it'll break if IPs change
-    resolver 8.8.8.8;
-    rewrite ^/_ouroboros_api(/.*) $1 break;
-    proxy_pass $ouroboros_api;
-    proxy_redirect $ouroboros_api /_ouroboros_api;
-}
-
-set $panoptes_api "https://panoptes.zooniverse.org";
-location /_panoptes_api/ {
```
Comment (adammcmaster) [inline nginx-api-proxy.conf:12]: Pretty sure we still want the Panoptes API proxy?

**Review by adammcmaster — CHANGES_REQUESTED** [review]

File: nginx-api-proxy.conf (hunk 1 above)
Comment (camallen) [reply nginx-api-proxy.conf:12]: i don't know any site that uses this as Panoptes has CORS configured correctly. Do we know of any?

Comment (adammcmaster) [reply nginx-api-proxy.conf:12]: I'm not sure actually... did we add this before we configured CORS then?

Comment (camallen) [reply nginx-api-proxy.conf:12]: my understanding of this was to tunnel the API requests via the same origin domain to api.zooniverse.org. I'm not sure why, maybe we didn't want to back port the clients? 

I don't think we ever used this as we setup the panoptes API to handle CORS requests and the client directly used the API host, either via cors or via a CDN like www.zooniverse.org 

GH Org level search doesn't show anything, https://github.com/search?q=org%3Azooniverse+_panoptes_api&type=Code

As opposed to ouroboros API code which was used to serve the API.js code to the client, https://github.com/search?q=org%3Azooniverse+_ouroboros_api&type=Code

Comment (camallen) [reply nginx-api-proxy.conf:12]: Ah - remembering vague details...While i wasn't really involved in this work, I believe we had to tunnel on the same domain to load the JS from the API via src html tag. Without this load from origin the JS would have been isolated to a different security context in the browser.

I'm not sure exactly why we load js from the API, but most likely a security isolation technique. I.e. if the API didn't respond on this route you didn't get access?

We don't use this technique for our JS SPA apps anymore and moving to SSR / static apps with next.js it's not really relevant.

Comment (adammcmaster) [reply nginx-api-proxy.conf:12]: OK cool, in that case we can go ahead and remove the remaining references to this file.

**Pull request closed without merging**